src/app/shared/components/postulant-credential/postulant-credential.component.html

```diff
@@ -4,13 +4,13 @@
   [width]="canvasWidth"
   [height]="canvasHeight"
 ></canvas>
-<h6 class="hide-on-med-and-up">{{ postulant.fullName }}</h6>
+<!-- <h6 class="hide-on-med-and-up">{{ postulant.fullName }}</h6> -->
```

Comment:
Just delete this, if we need it back we can always check the git history

Reply:
done!